app/pages/AppContainer.js

```diff
@@ -12,6 +13,7 @@ export class UnconnectedAppContainer extends Component {
   render() {
     return (
       <div className="app">
+        <Navbar />
         <DocumentTitle title="Huonevarausjärjestelmä" />
         {this.props.children}
```

Comment:
- [ ] Add a commit that wraps the children to react-bootstrap `Grid` so the page content is aligned with navbar content:
```javascript
<Grid>
  {this.props.children}
</Grid>
```